debian/patches/frr-reload-ldp.patch

```diff
@@ -1,7 +1,53 @@
+From: George Wilkie <george.wilkie@intl.att.com>
```

Comment:
Can you help me understand what is going on with this patch? There are a lot of new changes in this commit.

Reply:
Yes Sure.  
In the last commit we didn't check whether delete of configuration from DANOS CLI is reflecting in /etc/frr/frr.conf or not. However after checking that we found few issues, so we applied fix for it in this commit.

Reply:
Got it, thanks.

I wonder if all of these changes are required? Some of them (on first look) appear unrelated, for example the removal of the --bindir, --confdir etc. options. I will take another look tomorrow.

Ideally we'd keep the diff to upstream as small as possible, and also start pushing any necessary fixes to upstream frr.

Reply:
Hi, 
Many changes are reduced now on this "frr-reload-ldp.patch " patch. 
The  --bindir, --confdir etc. options are added back. There may be some indent related issues. 
Please have a look on this version of patch. 

Thanks, 
Kaushik

Reply:
Thanks Kaushik. After looking over the patch again, I think the only fix left is this:
```
diff --git a/tools/frr-reload.py b/tools/frr-reload.py
index fb8374bff..d7a2708fe 100755
--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -155,7 +155,7 @@ class Config(object):
         try:
             config_text = subprocess.check_output(
                 bindir + "/vtysh --config_dir " + confdir + " -c 'show run " + daemon + "' | /usr/bin/tail -n +4 | " + bindir + "/vtysh --config_dir " + confdir + " -m -f -",
-                shell=True)
+                shell=True, stderr=subprocess.STDOUT)
         except subprocess.CalledProcessError as e:
             ve = VtyshMarkException(e)
             ve.output = e.output
```
I'd suggest putting that in a new patch, and then removing frr-reload-ldp.patch.

Does that sound good? If so, please could you also squash this set of commits down into a single commit; then I think we can get this merged.

Reply:
Welcome !! Duncan. 
Yeah sure the change which you suggested is committed. So now only pending is squash this set of commits down into a single commit. 
Will do it and update you. 

Thanks,

Reply:
Hi, 
We have squashed this set of commits down into a single commit. 

Thanks,